xds/internal/xdsclient/authority.go

```diff
@@ -304,8 +334,35 @@ func (a *authority) newConnectionError(err error) {
 	// For all resource types, for all resources within each resource type, and
 	// for all the watchers for every resource, propagate the connection error
 	// from the transport layer.
```

Comment:
This comment is no longer appropriate here. I think we can delete it, what do you think?